E8G0NI/CourseManager/CourseManager/DTOs/CourseDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CourseManager.DTOs
+{
+    public class CourseDTO
```

Comment:
Ez inkább entity, nem DTO. DTO tipikusan REST API-nál van használva, mikor request json bejön és eredmény JSON kimegy. Ezek leírása a DTO ( = Data Transfer Object) Az entity meg ami a DB-be van. DB entity sosem DTO biztonsági okokból.